frontend/package.json

```diff
@@ -32,6 +32,7 @@
     "bnc-notify": "^1.5.1",
```

Comment:
*Arbitrary comment location*

As we get close to merging this, I was thinking it might be good to have an FAQ question. It can be under the "Sending Funds" heading, with the following copy:

#### Question

Why can't I see my send history on different devices?

#### Answer

On chain, the recipient of your sending transactions is not discernable. This is by design, and it is the point of Umbra—the receiving address is a stealth address. As a result, it's not possible to sync your send history across devices by observing on chain history.

Instead, your send history is encrypted and stored in your browser's local storage. It never leaves your device, and can't be seen on other devices as a result. Furthermore, transactions sent before this feature was added to the Umbra app in May 2023 will not be visible on any device.

Reply:
Added FAQ https://github.com/ScopeLift/umbra-protocol/pull/494/commits/f1d4f2a0b38884e09fbc6363589e8f6232f6626a
Added additional tex thttps://github.com/ScopeLift/umbra-protocol/pull/494/commits/bd98313c046eda17ffeb3435c47bf9423df19ef5

Reply:
Looks good, thanks!